WordPress/gutenberg
# Icons: Fix React warning error for offline icon

Related to #56934

## What?

This PR fixes the react warning error that appears when using the offline icon.

```
Warning: Invalid DOM property `fill-rule`. Did you mean `fillRule`?
Warning: Invalid DOM property `clip-rule`. Did you mean `clipRule`?
```
![icon](https://github.com/WordPress/gutenberg/assets/54422211/7e32c526-7aa7-413f-a7a3-895d4933aab2)

## How?

This can be solved by changing the attribute name to `fillRule`, `clipRule`, but since the icon did not change even without this attribute, I deleted it.

## Testing Instructions

- Make sure you don't get any browser warning errors when using code like the one below.
  ```javascript
  import { useBlockProps } from '@wordpress/block-editor';
  import { Icon, offline } from '@wordpress/icons';

  export default function Edit() {
  	const blockProps = useBlockProps();
  	return (
  		<div { ...blockProps }>
  			<Icon icon={ offline } />
  		</div>
  	);
  }
  ```
- The icon should remain the same in the mobile app.
